SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -799,11 +817,39 @@ void FSpatialGDKEditorToolbarModule::VerifyAndStartDeployment()
 	});
 }
 
-void FSpatialGDKEditorToolbarModule::StartSpatialDeploymentButtonClicked()
+void FSpatialGDKEditorToolbarModule::StartNoAutomaticConnectionButtonClicked()
+{
+	//Do nothing
+}
+
+void FSpatialGDKEditorToolbarModule::StartLocalSpatialDeploymentButtonClicked()
 {
 	VerifyAndStartDeployment();
 }
 
+void FSpatialGDKEditorToolbarModule::StartCloudSpatialDeploymentButtonClicked()
+{
+	// Create and open the cloud configuration dialog
+	SimulatedPlayerDeploymentWindowPtr = SNew(SWindow)
```

Comment:
This code should be moved into a function that's called both here and from `ShowSimulatedPlayerDeploymentDialog`. Also, we should probably update the naming of this window from "SimulatedPlayerDeployment" to just "CloudDeployment" (the original feature was focused around simulated players but this is no longer the case).

Reply:
I use ShowSimulatedPlayerDeploymentDialog() to replace it